scripts/airdrop.js

```diff
@@ -63,7 +57,7 @@ const parseTransferFile = async function (filename) {
       receiver,
       tokenAddress,
     }))
-  } else if (ext === "json") {
+  } else if (ext === ".json") {
     return JSON.parse(await fs.readFile(filename, "utf8"))
   } else {
     throw new Error(`unsupported file type .${ext}`)
```

Comment:
Just a little issue with my previous suggestion since `path.extname` already includes the `.`

```suggestion
    throw new Error(`unsupported file type ${ext}`)
```